docs/components/Navigation.js

```diff
@@ -21,80 +19,80 @@ function getAlphabetizedComponents() {
   );
 }
 
-function NavList({ sidebarOrganisedBy }: {| sidebarOrganisedBy: string |}) {
+function DocsSideNavigation(): Node {
+  const { sidebarOrganisedBy, setSidebarOrganizedBy } = useNavigationContext();
+  const router = useRouter();
+
   return (
-    <Fragment key={sidebarOrganisedBy}>
-      {sidebarOrganisedBy === 'categorized' ? (
-        sidebarIndex.map((section) => (
-          <SidebarSection section={section} key={`${sidebarOrganisedBy}-${section.sectionName}`} />
-        ))
-      ) : (
-        <Box marginTop={4}>
-          {getAlphabetizedComponents().map((componentName, i) => (
-            <SidebarSectionLink componentName={componentName} key={`${sidebarOrganisedBy}-${i}`} />
-          ))}
-        </Box>
-      )}
-    </Fragment>
+    <SideNavigation
+      accessibilityLabel="Side navigation"
+      header={
+        <nav label="Sidenav sort">
+          <SidebarCategorizationButton
```

Comment:
Could use a little extra top padding here above this button, it's a bit tight with the header. Maybe just surround in a box with some top margin

Reply:
@ayeshakmaz Let me check with hector, cos we discussed the top margin here.

Reply:
DONE